NitroxServer/ConsoleCommands/ListCommand.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxModel.Logger;
+
+namespace NitroxServer.ConsoleCommands
+{
+    class ListCommand : Command
+    {
+        private PlayerManager playerManager;
+
+        public ListCommand(PlayerManager playerManager)
+        {
+            Name = "list";
+            this.playerManager = playerManager;
+        }
+
+        public override void RunCommand(string[] args)
+        {
+            if (playerManager != null)
+            {
+                string list = string.Empty;
+                Player[] players = playerManager.GetPlayers().ToArray();
```

Comment:
Can be shortened to something like:

```CS
if (playerManager.GetPlayers().Any()) 
{
    Log.Info("Players: " + string.Join(playerManager.GetPlayers(), ", ")
}
else
{
    Log.Info("No players online");
}
```

Reply:
you seem to have flipped the args in string.Join() and missed the ; on the same line

Reply:
Yes, was doing it from memory.